openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -62,6 +62,9 @@
   text-align: right;
 }
 
+.text-underline {
+  text-decoration: underline;
+}
```

Comment:
As we are renaming the class `.underline` to `.text-underline` can we verify the same?

Reply:
Yes verified